fdbserver/fdbserver.actor.cpp

```diff
@@ -1593,6 +1593,11 @@ struct CLIOptions {
 } // namespace
 
 int main(int argc, char* argv[]) {
+	for (int i = 0; i < argc; ++i) {
+		std::cout << argv[i] << " ";
+	}
+	std::cout << std::endl;
```

Comment:
Is this intended to be printed unconditionally for everyone using fdbserver?

Reply:
Yes, it is. Given that this is one line per fdbserver invocation, the amount of noise is very small, and it is valuable to already have when needed.

Reply:
Sounds good, don't forget to include a quick mention of how this was tested in the description.

Reply:
Added that to the description. The build failure seems to be a flakey test, so I'll try to get them to re-run one way or another.

Reply:
FWIW there's no guarantee that if someone e.g. copied the command line invocation and pasted into a shell they'd get the same args because of shell quoting and whitespace etc.

Reply:
That's fair. If there is a better way of printing it out, I'm more than happy to do so. My experience has been that all fdbserver invocations are simple enough that this works well enough.

Reply:
I don't have a better suggestion - it was just a thought I had